mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -587,6 +578,15 @@ static void printParallelRegion(OpAsmPrinter &p, Operation *op, Region &region,
                               privateTypes, isByRef, privateSyms);
   }
 
+  if (reductionSyms) {
+    auto *argsBegin = region.front().getArguments().begin();
+    MutableArrayRef argsSubrange(argsBegin + privateVars.size(),
+                                 argsBegin + privateVars.size() +
+                                     reductionTypes.size());
```

Comment:
That's true, I just didn't bother here because the function is deleted in the following patch. Let me know if you'd prefer this to be updated anyways.

Reply:
Ahh good point. Yeah that is fine by me